test/fixtures/refunded-order-partial.json

```diff
@@ -0,0 +1,30 @@
+{
+  "input": {
+    "userId": "user-id",
+    "type": "track",
+    "event": "Order Refunded",
+    "properties": {
+      "orderId": 12345,
+      "products": [
```

Comment:
It isn't. Do we want to spec it? This might be a one-off for GA. It's worth
noting that our a.js GA integration does this already.

On Sunday, August 21, 2016, Prateek Srivastava <notifications@github.com>
wrote:

> In test/fixtures/refunded-order-partial.json
> <https://github.com/segment-integrations/integration-google-analytics/pull/48#discussion_r75617257>
> :
>
> > @@ -0,0 +1,30 @@
> > +{
> > +  "input": {
> > +    "userId": "user-id",
> > +    "type": "track",
> > +    "event": "Order Refunded",
> > +    "properties": {
> > +      "orderId": 12345,
> > +      "products": [
>
> products doesn't seem to be spec'd https://segment.com/docs/spec/
> ecommerce/v2/#order-refunded?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/segment-integrations/integration-google-analytics/pull/48/files/2771b369518609a1538b4d34f07b1da1688abf67#r75617257>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF-F7KVRn35rog7sS9T1GO2Mrn6cGGwYks5qiSfQgaJpZM4Jo8Ac>
> .
>


-- 
Sent from Mobile